presto-hive/src/main/java/io/prestosql/plugin/hive/rubix/RubixInitializer.java

```diff
@@ -43,6 +48,13 @@
  */
 public class RubixInitializer
 {
+    private static final RetryPolicy<?> RETRY_POLICY =
```

Comment:
The rest of the Hive connector uses `RetryDriver`, and I think we should stick with that, unless we cannot.  cc @electrum

Reply:
switcher to `RetryDriver`